fendermint/eth/api/src/apis/eth.rs

```diff
@@ -43,7 +46,12 @@ use crate::{
     error, JsonRpcData, JsonRpcResult,
 };
 
+// Lotus uses only 2 epochs to compupte the MAX_PRIO_FEE
+// but they compute the median over (on average) 10 blocks,
+// 5 per blocks. Thus, our 10 heights instead of 2.
+const NUM_BLOCKS_MAX_PRIO_FEE: u64 = 10;
 const MAX_FEE_HIST_SIZE: usize = 1024;
+const MIN_GAS_PREMIUM: u64 = 100_000;
```

Comment:
These could be moved into the `JsonRpcState` unders some struct, so they can be added to config instead of being hardcoded.

Reply:
Moved!